src/Game.h

```diff
@@ -99,6 +99,17 @@ class Game {
 	void SetTimeAccel(TimeAccel t);
 	void RequestTimeAccel(TimeAccel t, bool force = false);
 
+    /**
+     * Requests an increase in time acceleration
+     * @param force if set to false the system can reject the request under certain conditions
+     */
+    void RequestTimeAccelInc(bool force = false);
```

Comment:
Given that the rest of the file uses as line comments I would recommend sticking to that. I would also be happy for you to use 'three-slashes' line comments, which Doxygen also recognises. I.e.,

    /// Request an increase in time acceleration.
    /// @param force  If false, the actual acceleration level can be less than the requested level (e.g., due to nearby ships).